src/content/configuration/module.md

```diff
@@ -92,7 +92,7 @@ Specifies the category of the loader. No value means normal loader.
 
 There is also an additional category "inlined loader" which are loaders applied inline of the import/require.
 
-All loaders are sorted in the order `pre, inline, normal, post` and used in this order.
+All loaders are sorted in the order `pre, normal, inline, post` and used in this order.
```

Comment:
We should be explicit here about the order in someway. Maybe in a tooltop or hint annotation. 

Technically, webpack's loader execution goes from Left to Right to Left. 

So therefore in the first phase (the pitching phase), the order is: `post-inline-normal-pre`

Then for the next loader phase (the normal phase), the order is: `pre-normal-inline-post`

Reply:
What is the purpose of the pitching phase? 

Also, I think the explanation of the two phases needs to be added to [Writing a Loader](https://webpack.js.org/contribute/writing-a-loader/) section.

Reply:
I agree with Sean, maybe mention that there are two phases and then speak about order?